lib/sendgrid/mail.rb

```diff
@@ -56,5 +57,21 @@ def to_h
 
       payload
     end
+
+    private
+
+    def template
+      return if template_id.nil?
+
+      @template ||= Template.new(template_id)
+    end
+
+    def smtpapi_json
```

Comment:
No, since the `smtpapi` gem is used in `template.rb`, I might be inspect current filters on it but I'm not sure. I can check though.